migrations/automatic/educandu-2022-01-10-01-update-task-and-batch-type-for-imports.js

```diff
@@ -0,0 +1,18 @@
+import { BATCH_TYPE, TASK_TYPE } from '../../src/domain/constants.js';
+
+// eslint-disable-next-line camelcase
+export default class Educandu_2022_01_10_01_update_task_and_batch_type_for_imports {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.collection('batches').updateMany({ batchType: 'import-documents' }, { $set: { batchType: BATCH_TYPE.documentImport } });
+    await this.db.collection('tasks').updateMany({ taskType: 'import-document' }, { $set: { taskType: TASK_TYPE.documentImport } });
+  }
+
+  async down() {
+    await this.db.collection('batches').updateMany({ batchType: BATCH_TYPE.documentImport }, { $set: { batchType: 'import-documents' } });
```

Comment:
This down won't really work if we change the value of the constants but I think we will remove the migration soon anyway.

Reply:
Exactly what I wanted to mention, that it would be better to just use hardcoded strings in the migration scripts and not reference any code from outside.

Reply:
Idempotent, that's a word I haven't used in a while 😋